bundles/org.openhab.core.compat1x/src/main/java/org/openhab/io/net/exec/ExecUtil.java

```diff
@@ -137,7 +137,7 @@ public static String executeCommandLineAndWaitResponse(String commandLine, int t
                 logger.debug("exit code '{}', result '{}'", exitCode, retval);
             }
         } catch (InterruptedException e) {
-            logger.error("Timeout occured when executing commandLine '" + commandLine + "'", e);
+            logger.error("Timeout occurred when executing commandLine '" + commandLine + "'", e);
```

Comment:
The reason nothing fired over this is that it is an logger.error so it will most likely always be visible and thus the load is also not extra, I agree on changing it.

I will include the lines a little big higher.

Note that ExecuteException is an instance of IOException so I also simplified the catch.